storage_backend_ftp/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
the license of the module is LGPL. Can you check all license references in this module pls?

Reply:
Hi,

Updated License details.

Thanks

Reply:
can you make a run on v14 too?